c2cgeoportal/scaffolds/update/CONST_packages.yaml

```diff
@@ -53,6 +31,45 @@ docker:
         cmd: docker-compose --version | awk '{{print $3}}'
         version: 1.4.2
 
+mapserver:
+    mapserver:
+        cmd: /usr/lib/cgi-bin/mapserv -v | awk '{{print $3}}'
+        version: 7.0
+
+postgres:
+    postgres:
+        cmd: psql --version | awk '{{print $3}}'
+        version: 9.1
+    postgis:
+        cmd: sudo -u postgres psql template_postgis -c "select PostGIS_lib_version();" -At
+        #cmd: dpkg -l | grep postgis | grep ^ii | awk '{{print $3}}' | head -n 1
+        version: 2.0
+
+print:
+    java:
+        cmd: java -version 2>&1 | grep  -E '(java|openjdk) version' | awk '{{print $3}}' | sed 's/"//g' | sed 's/_/./g'
+        version: 1.7.0
+    tomcat:
+        cmd: /usr/share/tomcat*/bin/version.sh | grep 'Server number' | awk '{{print $3}}'
+        version: 6.0
+    unzip:
+        cmd: unzip -v | head -n1 | awk '{{print $2}}'
+        version: 6.0
+    zip:
+        cmd: zip -v | head -n2 | tail -n1 | awk '{{print $4}}'
+        version: 3.0
+
+docker-print:
```

Comment:
You don't need java, zip, unzip to do print with docker. Docker is all you need.